src/Glpi/Controller/InventoryController.php

```diff
@@ -0,0 +1,121 @@
+<?php
+
+/**
+ * ---------------------------------------------------------------------
+ *
+ * GLPI - Gestionnaire Libre de Parc Informatique
+ *
+ * http://glpi-project.org
+ *
+ * @copyright 2015-2024 Teclib' and contributors.
+ * @licence   https://www.gnu.org/licenses/gpl-3.0.html
+ *
+ * ---------------------------------------------------------------------
+ *
+ * LICENSE
+ *
+ * This file is part of GLPI.
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <https://www.gnu.org/licenses/>.
+ *
+ * ---------------------------------------------------------------------
+ */
+
+namespace Glpi\Controller;
+
+use Glpi\Exception\Http\AccessDeniedHttpException;
+use Glpi\Http\Firewall;
+use Symfony\Component\HttpFoundation\Request;
+use Glpi\Inventory\Conf;
+use Glpi\Security\Attribute\SecurityStrategy;
+use Symfony\Component\HttpFoundation\RedirectResponse;
+use Symfony\Component\HttpFoundation\Response;
+use Symfony\Component\Routing\Attribute\Route;
+
+final class InventoryController extends AbstractController
+{
+    #[SecurityStrategy(Firewall::STRATEGY_NO_CHECK)]
+    #[Route("/Inventory", name: "glpi_inventory")]
+    public function __invoke(Request $request): Response
+    {
+        $conf = new Conf();
+
+        if ($conf->enabled_inventory != 1) {
+            throw new AccessDeniedHttpException("Inventory is disabled");
+        }
+
+        $inventory_request = new \Glpi\Inventory\Request();
+        if ($inventory_request->inError() && $inventory_request->getHttpResponseCode() == 415) {
+            http_response_code($inventory_request->getHttpResponseCode());
+            die("Unsupported compression");
+        }
```

Comment:
This is not required anymore as you now throw an exception for this.

Reply:
Yep, it's now dropped